po/de.po

```diff
@@ -30,7 +30,7 @@ msgid "Addon"
 msgstr "Addon"
 
 #: gtk3/main_window.c:283
-msgid "Advance"
+msgid "Advanced"
 msgstr "Fortschritt"
```

Comment:
According to some dictionaries, English noun "advance" can be translated as German "Fortschritt" ([[1]](https://en.wiktionary.org/wiki/advance), [[2]](https://translate.google.com/#en/de/advance)), while English adjective "advanced" can be translated as German "fortgeschritten" ([[3]](https://en.wiktionary.org/wiki/advanced), [[4]](https://translate.google.com/#en/de/advanced)).